internal/gatewayapi/testdata/gateway-allows-same-namespace-with-allowed-httproute.out.yaml

```diff
@@ -71,7 +71,7 @@ xdsIR:
 infraIR:
   proxy:
     name: envoy-gateway-class
-    image: envoyproxy/envoy-dev:latest
+    image: envoyproxy/envoy:v1.23-latest
```

Comment:
We should find a way to not have to bump these test cases every time we change the Envoy version, but fine to put on the backlog for now.